More than one geojson

**Please complete the following information**
[Data_config](https://github.com/WohnungNDS/IM_Data)
Geojsonfile1 : [NI_Kurz_diss.geojson](https://github.com/WohnungNDS/IM_Data/blob/develop/NI_Kurz_diss.geojson)
Geojsonfile2: [world_custom.geojson](https://github.com/WohnungNDS/IM_Data/blob/develop/world_custom.geojson)

Data or metadata issue: Data/Map/GeoJson
Input format: CSV, Geojson

**Describe the bug**
I have two different geojson files in my datarepo. 
geojsonfile1 covers the administrative local boundaries of lower saxony,
geojsonfile2 covers the boundaries off all states over the world.

The idea is to show where people from all over the world come to lower saxony. eg 1234 from Poland 1222 from Turkey tbc

I add in my data_config file this:

`map_layers:
  - geojson_file: NI_Kurz_diss.geojson
    name_property: GEN
    id_property: AGS
    output_subfolder: maps
    filename_prefix: indicator_
    exclude_columns: ['Gebietseinheit']
  - geojson_file: world_custom.geojson
    name_property: name_long
    id_property: LAGS
    output_subfolder: maps
    filename_prefix: indicator_
`
And added some test data to indicator [Data 4.1.3](https://github.com/WohnungNDS/IM_Data/blob/develop/data/indicator_4-1-3.csv) 

The map is shown more ore less correctly as you can see [here](https://wohnungnds.github.io/IM_Site/4-1-3/)

But now the other maps eg in [indicator 1.1.1](https://wohnungnds.github.io/IM_Site/1-1-1/) contains no boundaries an also no data. There should be shown boundaries from Geojsonfile1 and data form indicator 1.1.1

Do I have to do changes in my site [_config](https://github.com/WohnungNDS/IM_Site/blob/develop/_config.yml) file too?
Any Ideas?

@LucyGwilliamAdmin  Thank you I have fixed this....
But there are more promlems with my geojson files. to me it is not clear how SDG decides which Goejson file schould be used.....
Sometimes I get Errors like this.... But I have no idea what this error means



`  File "/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/site-packages/sdg/outputs/OutputGeoJson.py", line 148, in <listcomp>
    disaggregations = [self.clean_disaggregations(disaggregation) for disaggregation in disaggregations]
  File "/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/site-packages/sdg/outputs/OutputGeoJson.py", line 214, in clean_disaggregations
    del disaggregations[self.id_column]
KeyError: 'GeoCode'
Error: Process completed with exit code 1.`

--- a/docs/configuration.md
+++ b/docs/configuration.md
@@ -765,6 +765,14 @@ logos:
     alt: mi texto alternativo
 ```
 
+### map_layers
+
+_Optional_: This setting configures the layers that will be visible on maps on indicator pages. Each layer is a set of boundaries. For example, one layer might be for the province boundaries, and another layer might be for the district boundaries. For more information on this setting, see the [Maps guidance](maps.md).
+
+### map_options
+
+_Optional_: This setting configures general options for maps on indicator pages. For more information on this setting, see the [Maps guidance](maps.md).
+
 ### metadata_edit_url
 
 **_Required_**: This setting controls the URL of the "Edit Metadata" that appear on the staging site's indicator pages. It should be a full URL. Note that you can include `[id]` in the URL, and it will be dynamically replaced with the indicator's id (dash-delimited).

--- a/docs/maps.md
+++ b/docs/maps.md
@@ -108,10 +108,15 @@ map_layers:
     # The label to use in the "Download GeoJSON" button:
     label: indicator.map
     # The minimum zoom at which this layer should be visible.
+    # For example, if this "min_zoom" is set to 4, then this
+    # layer will be invisible at zoom level 3 or less.
     min_zoom: 0
     # The maximum zoom at which this layer should be visible.
+    # For example, if this "max_zoom" is set to 8, then this
+    # layer will be invisible at zoom level 9 or more.
     max_zoom: 20
-    # Whether or not these boundaries should display statically on lower layers.
+    # Whether or not these boundaries should display statically
+    # on lower layers.
     staticBorders: false
 ```
 

